java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -73,4 +74,30 @@ public interface StreamBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> xtrim(String key, StreamTrimOptions options);
+
+    /**
+     * Reads entries from the given streams.
+     *
+     * @see <a href="https://redis.io/commands/xread/">redis.io</a> for details.
+     * @param keysAndIds - A <code>Map</code> of keys and entry ids to read from. The <code>
```

Comment:
remove dashes thoughout
```suggestion
     * @param keysAndIds A <code>Map</code> of keys and entry ids to read from. The <code>
```